### PR TITLE
Update license template.

### PR DESCRIPTION
**Pull request by ekpyron**
Someone should properly double-check this

### CODE REVIEW

File: cmake/templates/license.h.in [hunk 1]
```diff
@@ -5,12 +5,69 @@
...
+  copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+  SOFTWARE.
+
```
Comment (matheusaaguiar) [inline cmake/templates/license.h.in:33]: The [license](https://json.nlohmann.me/home/license/) at the official website (?) has some more paragraphs pointing to other contributions and their respective licenses. Not sure if we should include that also.
```
The class contains the UTF-8 Decoder from Bjoern Hoehrmann which is licensed under the [MIT License](https://opensource.org/licenses/MIT) (see above). Copyright © 2008-2009 [Björn Hoehrmann](http://bjoern.hoehrmann.de/) [bjoern@hoehrmann.de](mailto:bjoern@hoehrmann.de)

The class contains a slightly modified version of the Grisu2 algorithm from Florian Loitsch which is licensed under the [MIT License](https://opensource.org/licenses/MIT) (see above). Copyright © 2009 [Florian Loitsch](https://florian.loitsch.com/)

The class contains a copy of [Hedley](https://nemequ.github.io/hedley/) from Evan Nemerson which is licensed as [CC0-1.0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment (ekpyron) [reply cmake/templates/license.h.in:33]: Yeah, I just copied https://github.com/nlohmann/json/blob/develop/LICENSE.MIT but https://json.nlohmann.me/home/license/ is indeed better.

Comment (ekpyron) [reply cmake/templates/license.h.in:33]: Updated it

**Review by matheusaaguiar — DISMISSED** [review]

File: cmake/templates/license.h.in [hunk 2]
```diff
@@ -34,62 +85,161 @@ picosha2:
...
+  //
+  // Permission to use, copy, modify, distribute and sell this software
+  // and its documentation for any purpose is hereby granted without fee,
+  // provided that the above copyright notice appear in all copies and
+  // that both that copyright notice and this permission notice appear
+  // in supporting documentation.  Silicon Graphics makes no
+  // representations about the suitability of this software for any
+  // purpose.  It is provided "as is" without express or implied warranty.
+  //
+
```
Comment (ekpyron) [inline cmake/templates/license.h.in:211]: This is actually a superfluous line break, I'll still remove it.

Comment (ekpyron) [reply cmake/templates/license.h.in:211]: Done.

**Review by matheusaaguiar — DISMISSED** [review]

**Review by matheusaaguiar — APPROVED** [review]

File: cmake/templates/license.h.in [hunk 3]
```diff
@@ -3,14 +3,34 @@
 #include <string>
 
 static std::string const otherLicenses{R"(Most of the code is licensed under GPLv3 (see below), the license for individual
-parts are as follows:
+parts and dependencies are as follows:
```
Comment (ibrahimkhled) [inline cmake/templates/license.h.in:6]: acec7170bad18b3dce83badec128e3cb75caec53

**Pull request merged**